corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserEditor.java

```diff
@@ -408,8 +408,9 @@ public CorfuDynamicRecord addRecord(String namespace, String tableName,
             dynamicProtobufSerializer.createDynamicMessageFromJson(defaultValueAny,
                 newValue);
         DynamicMessage newMetadataMsg =
```

Comment:
instead of adding this validation, can we directly set newMetadataMsg to null if the input argument is null?

This did not occur to me yesterday but if you agree with the approach, it will make the change simpler.

Reply:
Doesn't that change the semantics of what this `addRecord()` does - currently if the table had metadata configured then we will expect a metadata to be passed in. But now if we just validate it on the incoming metadata (being null or not) it means the users can pass null metadata in a table which had non-null metadata configured right?

Reply:
discussed offline - implemented the validation to fail if table had metadata configured but user attempts to pass null.